test/glue/candlepin/consumer_test.rb

```diff
@@ -11,7 +11,7 @@ class GlueCandlepinConsumerTestBase < ActiveSupport::TestCase
     @@dev_cv = nil
     @@dev_cve = nil
 
-    def self.before_suite
+    def setup
```

Comment:
Does this still need the super call? If we are doing a `setup` call instead of a suite wrapper, I would think the class variables would not be needed ?